src/ResourceManager/DeploymentManager/Commands.DeploymentManager/AzureRM.DeploymentManager.psd1

```diff
@@ -0,0 +1,155 @@
+#
+# Module manifest for module 'PSGet_AzureRM.DeploymentManager'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/25/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.9.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'caac1590-e859-444f-a9e0-62091c0f5929'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Deployment Manager cmdlets for Azure Resource Manager.  Creates and manages Deployment Manager resources in Azure Resource Manager.'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '5.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.1'; })
```

Comment:
`5.6.0` is the version of profile to use now : https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Aks/Commands.Aks/AzureRM.Aks.psd1#L54